examples/StandardFirmataEthernet/ethernetConfig.h

```diff
@@ -55,7 +56,9 @@ YunClient client;
 
 // STEP 3 [REQUIRED]
 // Replace with the port that your server is listening on
+// for client mode comment out remote_port / for server mode comment out server_port
 #define remote_port 3030
```

Comment:
This could be simplified by renaming `remote_port` to `network_port` and then using that for both client and server applications.